pkg/secrets/test.go

```diff
@@ -0,0 +1 @@
+package secrets
```

Comment:
Is this like Python's `__init__.py` file?

Reply:
No, thats a mistake I think.